deploy/kwok/setup.sh

```diff
@@ -0,0 +1,138 @@
+: ${NUODB_CP_USER:="system/admin"}
+: ${NUODB_CP_PASSWORD:="changeIt"}
+
+echo "Creating K8s cluster..."
+kwokctl create cluster --wait 1m
+kwokctl scale node --replicas 5
+
+CLUSTER_DIR="$KWOK_WORKDIR/clusters/kwok"
+DOCKER_NET="kwok-kwok"
+
+# Make everything in kwok directory world-readable to avoid annoying Docker
+# volume permissions issues. The NuoDB CP image has uid 1000 for the runtime
+# user and would not be able access files owned by root on the host that are
+# not group- or world-readable. In CircleCI, the root user invokes this script,
+# and by default the K8s credentials are not world-readable.
+chmod -R a+X,a+r "$CLUSTER_DIR"
+
+# Start noop volume provisioner that immediately binds PVCs
+docker run -d --name noop-provisioner \
+    -v "$CLUSTER_DIR"/kubeconfig:/kubeconfig \
+    -v "$CLUSTER_DIR"/pki:/etc/kubernetes/pki \
+    --network "$DOCKER_NET" \
+    "$PROVISIONER_IMAGE" -kubeconfig /kubeconfig
+
+# Create storage class for noop provisioner
+kubectl apply -f - <<EOF
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: noop
+provisioner: nuodb.github.io/noop-provisioner
+EOF
+
+echo "Installing CRDs for DBaaS..."
+helm install nuodb-cp-crd nuodb-cp-crd --repo "$NUODB_CP_REPO" --version "$NUODB_CP_VERSION"
+
+# Create service tiers. The Helm features do not matter, since they are not
+# exposed in Terraform.
+kubectl apply -f - <<EOF
+apiVersion: cp.nuodb.com/v1beta1
+kind: ServiceTier
+metadata:
+  name: n0.nano
+spec:
+  features: []
+---
+apiVersion: cp.nuodb.com/v1beta1
+kind: ServiceTier
+metadata:
+  name: n0.small
+spec:
+  features: []
+---
+apiVersion: cp.nuodb.com/v1beta1
+kind: ServiceTier
+metadata:
+  name: n1.small
+spec:
+  features: []
+EOF
+
+# Create service account needed for AP
+kubectl create serviceaccount nuodb
+
+# Start Operator with webhooks and backups disabled, since they cannot work in
+# this environment
+echo "Starting DBaaS Operator..."
+docker run -d --name nuodb-cp-operator \
+    -v "$CLUSTER_DIR"/kubeconfig:/home/nuodb/.kube/config \
+    -v "$CLUSTER_DIR"/pki:/etc/kubernetes/pki \
+    -e ENABLE_WEBHOOKS=false \
+    --network "$DOCKER_NET" \
+    "$NUODB_CP_IMAGE" \
+    controller --feature-gates EmbeddedDatabaseBackupPlugin=false
+
+echo "Starting DBaaS REST service..."
+docker run -d --name nuodb-cp-rest -p 8080:8080 \
+    -v "$CLUSTER_DIR"/kubeconfig:/home/nuodb/.kube/config \
+    -v "$CLUSTER_DIR"/pki:/etc/kubernetes/pki \
+    --network "$DOCKER_NET" \
+    "$NUODB_CP_IMAGE" \
+    nuodb-cp server start
+
+echo "Waiting for REST service to become ready..."
+check() {
+    out="$(docker exec nuodb-cp-rest nuodb-cp httpclient GET /healthz 2>&1)"
+}
+i=0
+n=15
+while ! check; do
+    if [ $i -ge $n ]; then
+        echo "Readiness check failed after $n seconds:"
+        echo "$out"
+        exit 1
+    fi
+    echo "Retrying in 1 second..."
```

Comment:
`Retrying in 1 second...` -> `Retrying in $i seconds...`

Reply:
Will do.

Reply:
Actually, I screwed this up. I actually wanted to poll every second instead of increasing the period.